wiki/en/en-Installation-for-Windows.md

```diff
@@ -41,15 +41,16 @@ Before you start with Jamulus:
 ### How to set up ASIO4ALL inputs (Guide)
 
 1. Open Jamulus's settings
-1. Go to _"ASIO Settings"_ (column on the left)
+1. Go to _"ASIO Device Settings"_ (column on the left; directly under the selection of the driver)
 1. Enable _advanced view_ in ASIO4ALL (click the tool icon on the bottom right)
 1. Enable only the sound card you want to use by clicking on the button next to its name
 1. Open your sound card inputs/outputs by clicking the _plus icon_ next to this sound card
-1. Now enable the correct inputs/outputs in the list under your sound card and disable everything else
+1. Now enable the correct inputs/outputs in the list under your sound card and disable everything else. You can hover over the inputs/outputs to see which of both they are and if they support the required sample rate for Jamulus of 48kHz (DVD quality).
 
-**Hint:** Many internal sound cards are named Realtek High Definition Audio, Conexant or similar.
-
-Concerning the inputs/outputs: Headphones and speakers are often labelled as output and microphones as inputs or capture. Stereo Mix is usually not the input/output you're looking for and therefore you can switch it off if you see it.
+**Hints:**
+1. Many internal sound cards are named Realtek High Definition Audio, Conexant or similar.
+1. Headphones and speakers are often labelled as output and microphones as inputs or capture.
+1. Stereo Mix is usually not the input/output you're looking for and therefore you can switch it off if you see it.
```

Comment:
```suggestion
1. Stereo Mix is usually not the input/output you're looking for and therefore you should switch it off if you see it.
```

In my experience after onboarding my choir: Stereo mix has been the reason for getting no input (or worse: feedback!), so I would be more explicit suggest disabling it.

Reply:
Ok. Added your suggestion. Can still be re-thought.